gpx.go

```diff
@@ -70,15 +72,15 @@ type Wpt struct {
 
 // Rte is a GPX Route
 type Rte struct {
-	XMLName     xml.Name `xml:"rte"`
-	Name        string   `xml:"name,omitempty"`
-	Cmt         string   `xml:"cmt,omitempty"`
-	Desc        string   `xml:"desc,omitempty"`
-	Src         string   `xml:"src,omitempty"`
-	Links       []Link   `xml:"link"`
-	Number      int      `xml:"number,omitempty"`
-	Type        string   `xml:"type,omitempty"`
-	RoutePoints []Wpt    `xml:"rtept"`
+	XMLName   xml.Name `xml:"rte"`
+	Name      string   `xml:"name,omitempty"`
+	Cmt       string   `xml:"cmt,omitempty"`
+	Desc      string   `xml:"desc,omitempty"`
+	Src       string   `xml:"src,omitempty"`
+	Links     []Link   `xml:"link"`
+	Number    int      `xml:"number,omitempty"`
+	Type      string   `xml:"type,omitempty"`
+	Waypoints `xml:"rtept"`
```

Comment:
Same as Trkseg. We get Gpx.Routes[0].Length3D() instead of Gpx.Routes[0].RoutePoints.Length3D().

That was the motivation, I can change them back if you prefer it the other way.